File: C#/loops.cs
using System;

namespace Loops {
    public class App {
        static void Main (string[] args) {
            for (int j = 1; j < 5; j++) {
                Console.WriteLine("for loop iteration {0}", j);
            }

            int i = 1;
            while (i < 5) {
                Console.WriteLine("while loop iteration {0}", i);
                i++;
            }

            i = 1;
            do {
                Console.WriteLine("do while loop iteration {0}", i);
                i++;
            } while (i < 5);
        }
    }
}

File: Java/Loops.java
public class Loops {
    public static void main(String[] args) {
        for (int i = 1; i < 5; i++) {
            System.out.println("for loop iteration " + i);
        }

        int i = 1;
        while (i < 5) {
            System.out.println("while loop iteration " + i);
            i++;
        }

        i = 1;
        do {
            System.out.println("do while loop iteration " + i);
            i++;
        } while (i < 5);
    }
}